content/tutorials/provision-dashboards-and-data-sources.md

```diff
@@ -113,15 +116,16 @@ providers:
     options:
       path: /var/lib/grafana/dashboards # The path to the dashboard definitions
 ```
+((I do not have this path in my Grafana installation on my Windows machine. This is my path: C:\Program Files\GrafanaLabs\grafana\public\dashboards))
 
 For more information on how to configure dashboard providers, refer to [Dashboards](https://grafana.com/docs/grafana/latest/administration/provisioning/#dashboards).
 
 #### Create a dashboard definition
 
-- In the dashboard definitions directory you specified in the dashboard provider, i.e. `options.path`, create a file called `cluster.json` with the following content:
+1. In the dashboard definitions directory you specified in the dashboard provider, i.e. `options.path`, create a file called `cluster.json` with the following content:
```

Comment:
Again, this will reset numbering after code block.

Reply:
I tried hard coding the numbers. Please let me know if it works. If not, I will change the list back to an unordered one.